bigbluebutton-html5/private/config/settings.yml

```
@@ -63,6 +63,7 @@ public:
     allowUserLookup: false
     dynamicGuestPolicy: true
     enableGuestLobbyMessage: true
+    guestPolicyExtraAllowOptions: false
```

```suggestion
    guestPolicyExtraAllowOptions: true
```

let's have the default value preserve the current view